src/controllers/expenses.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+const { expensesService } = require('../services/expensesService.js');
+const { userService } = require('../services/usersService.js');
+
+const getAll = (req, res) => {
+  const filteredExpenses = expensesService.getAll(req.query);
+
+  res.statusCode = 200;
+  res.send(filteredExpenses);
+};
+
+const getById = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expensesService.getById(Number(expenseId));
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
+  }
+
+  res.statusCode = 200;
+  res.send(foundExpense);
+};
+
+const create = (req, res) => {
+  const {
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  } = req.body;
+
+  const isUserExist = userService.getById(Number(userId));
+
+  if (!title || !isUserExist) {
+    res.sendStatus(400);
+
+    return;
+  }
+
+  const newExpense = expensesService.create(
+    userId,
```

Comment:
Use destructuring assignment for the arguments passed to the create function to make the code more readable and less prone to errors.